[3.6.0] Cannot find 'Request'

Hello,

After upgrading to 3.6.0 a typescript error popped-up.
Here's the output of the tsc command.

```
tsc --noEmit

node_modules/fastify-gql/index.d.ts:230:40 - error TS2304: Cannot find name 'Request'.
230   transformRequest?: (originalRequest: Request) => Request;
                                           ~~~~~~~

node_modules/fastify-gql/index.d.ts:230:52 - error TS2304: Cannot find name 'Request'.
230   transformRequest?: (originalRequest: Request) => Request;
                                                       ~~~~~~~
```

Looking to the index.d.ts file, no Request type seems to be imported or declared or aliased.

The issue was introduced by this pull request : https://github.com/mcollina/fastify-gql/pull/149

--- a/index.d.ts
+++ b/index.d.ts
@@ -255,3 +255,9 @@ type Result = ExecutionResult & {
 interface IGraphQLToolsResolveInfo extends GraphQLResolveInfo {
   mergeInfo?: MergeInfo;
 }
+
+type Request = {
+  document: DocumentNode;
+  variables: Record<string, any>;
+  extensions?: Record<string, any>;
+}; 

